docs/source/troubleshooting.md

```diff
@@ -152,6 +193,29 @@ After this, when you start your server via JupyterHub, it will build a
 new container. If this was the underlying cause of the issue, you should see
 your server again.
 
+### Launching Jupyter Notebooks to run as an externally managed JupyterHub service with the `jupyterhub-singleuser` command returns a `JUPYTERHUB_API_TOKEN` error.
+
+If possible, try to run the Jupyter Notebook as an externally managed service with one of the provided [jupyter/docker-stacks](https://github.com/jupyter/docker-stacks).
```

Comment:
When do you start a notebook as a service? Asking because I"ve never heard people do this or done it myself so when I first read the title I thought it was a typo. Maybe including that use-case explicitly helps people who are doing that to recognise this entry and will help to not lead people who don't want  to be doing this astray. What do you think?

Reply:
Some folks do this for a 'shared' notebook environment, while JupyterHub still lacks support for projects/shared workspaces. I've seen it in nbgrader deployments, where all graders have access to a single notebook server with nbgrader stuff set up.

I think this use case will probably go away when the concept of shared projects finally arrives.